src/settings.js

```diff
@@ -999,7 +999,7 @@ var INCOMING_MODULE_JS_API = [
   'SDL_numSimultaneouslyQueuedBuffers', 'INITIAL_MEMORY', 'wasmMemory', 'arguments',
   'buffer', 'canvas', 'doNotCaptureKeyboard', 'dynamicLibraries',
   'elementPointerLock', 'extraStackTrace', 'forcedAspectRatio',
-  'instantiateWasm', 'keyboardListeningElement', 'freePreloadedMediaOnUse',
```

Comment:
New imports should instead get added to `EXTRA_INCOMING_JS_API` in `emcc.py` to avoid adding bloat to all programs by default.